### PR TITLE
chore: update videoCompression config values to reflect new options.

### PR DESCRIPTION
**Pull request by AtofStryker**
corresponding docs change for [#26810](https://github.com/cypress-io/cypress/pull/26810) where `videoCompression` is valid value between `1`-`51` and can now be `true`, which defaults to `32` CRF. `0` or `false` disables compression

### CODE REVIEW

**Comment by AtofStryker** [comment]
I think github might be having issues on PRs syncing branches on update. You can see the changes are reflected in the branch https://github.com/cypress-io/cypress-documentation/blob/bugfix/video_compression_values/docs/guides/references/configuration.mdx#videos 🤷🏻

**Comment by MikeMcC399** [comment]
@AtofStryker 

See https://www.githubstatus.com/ which is reporting an issue with GitHub operations right now.

File: docs/guides/references/configuration.mdx [hunk 1]
```diff
@@ -124,7 +124,7 @@ For more options regarding screenshots, view the
...
 | `trashAssetsBeforeRuns` | `true`           | Whether Cypress will trash assets within the `downloadsFolder`, `screenshotsFolder`, and `videosFolder` before tests run with `cypress run`.                                                                                                                                                                             |
-| `videoCompression`      | `32`             | The quality setting for the video compression, in Constant Rate Factor (CRF). The value can be `false` to disable compression or a value between `0` and `51`, where a lower value results in better quality (at the expense of a higher file size).                                                                     |
+| `videoCompression`      | `32`             | The quality setting for the video compression, in Constant Rate Factor (CRF). The value can be `false` to disable compression or a value between `0` and `51`, where a lower value results in better quality (at the expense of a higher file size). Setting this option to `true` will result in a default CRF of `32`.                                                                 |
```
Comment (emilyrohrbough) [inline docs/guides/references/configuration.mdx:127]: can we align with recent description updates on the code pr?

Comment (AtofStryker) [reply docs/guides/references/configuration.mdx:127]: yep taking care of that right now. should be up to date in a few minutes

**Comment by AtofStryker** [comment]
@emilyrohrbough this should be ready for another pass
Screenshot 2023-06-08 at 12 31 57 PM

**Review by emilyrohrbough — APPROVED** [review]

File: docs/guides/references/configuration_legacy.mdx [hunk 2]
```diff
@@ -101,13 +101,13 @@ For more options regarding screenshots, view the
...
+| ----------------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `trashAssetsBeforeRuns` | `true`           | Whether Cypress will trash assets within the `downloadsFolder`, `screenshotsFolder`, and `videosFolder` before tests run with `cypress run`.                                                                                                                                                                                  |
+| `videoCompression`      | `32`             | The quality setting for the video compression, in Constant Rate Factor (CRF). The value can be `false` or `0` to disable compression or a CRF between `1` and `51`, where a lower value results in better quality (at the expense of a higher file size). Setting this option to `true` will result in a default CRF of `32`. |
```
Comment (emilyrohrbough) [inline docs/guides/references/configuration_legacy.mdx:107]: This should not have been update. This is the legacy configuration for <v 9 and this isn't valid for these versions.

**Review by emilyrohrbough — CHANGES_REQUESTED** [review]

File: docs/guides/references/configuration_legacy.mdx (hunk 2 above)
Comment (AtofStryker) [reply docs/guides/references/configuration_legacy.mdx:107]: OK I'll rebase it out

Comment (AtofStryker) [reply docs/guides/references/configuration_legacy.mdx]: rebased out

**Pull request merged**